components/leaflet-maps/ClustersMap.vue

```diff
@@ -115,6 +120,12 @@
          detention: "Detention",
          placeOfDeath: "Place of Death"
        },
+       standardAttributes:["sex","occupation","socialClass"],
+       standardLabels:{
```

Comment:
I would probably rename this StandardAttributeLabels. Just so it is a little bit clearer.